### DISCUSSION TITLE
Unable to start GPU-assisted computing

### DISCUSSION THREAD

**Discussion by RobinChen3** [securefederatedai/openfederatedlearning#447] [Q&A]
We are testing the TinyImageNet example, with a GPU card (Nvidia 1050ti) installed on the host, and a value set in cuda_devices:[0] in envoy_config.yaml. However, still can't start GPU assisted computing, any suggestions?

**Comment by mansishr** [comment]
Hi @RobinChen3 , could you try `nvidia-smi` to check the list of available GPUs? Also, if you are trying out [this example](https://github.com/intel/openfl/blob/develop/openfl-tutorials/interactive_api/PyTorch_TinyImageNet/workspace/pytorch_tinyimagenet.ipynb), the device is explicitly set to `'cpu'` in `train `and `validate `tasks so you must comment those lines.
